Source/views/profile.hbs

```diff
@@ -0,0 +1,39 @@
+<!-- Profiel Pagina -->
```

Comment:
Niet in lijn met het ontwerp van de rest van het project. De knop is oranje (maak deze blauw) en de tekst staat los op de pagina i.p.v. netjes in een blok, zie als voorbeeld de admin pagina.